RulesEngine/RuleProcessor.cs

```diff
@@ -216,28 +214,57 @@ void ProcessBoundary(ClauseCapture cap)
             return resultsList.ToArray();
         }
 
-  
+
         #region Private Support Methods
 
         /// <summary>
         ///     Filters the rules for those matching the content type. Resolves all the overrides
         /// </summary>
         /// <param name="languages"> Languages to filter rules for </param>
         /// <returns> List of rules </returns>
-        private IEnumerable<ConvertedOatRule> GetRulesForSingleLanguage(string language)
+        private IEnumerable<ConvertedOatRule> GetRulesByLanguage(string input)
+        {
+            string langid = string.Empty;
+
+            if (EnableCache)
+            {
+                // Make language id for cache purposes
+                langid = string.Join(":", input);
```

Comment:
I don't think this does anything.

Reply:
```suggestion
                langid = string.Join(":", input);
```

This isn't actually new code. But yes, I don't know why that's being appended. I'll remove it.